app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1514,3 +1535,293 @@ table li .account_details {
+}
+
+.item-show-description {
+  border: 1px solid red;
+  font-family: 'Cardinal', Arial, sans-serif;
+  font-size: 48px;
+  margin: 10px 0;
+  padding: 0px 5px;
+}
+
+.item-categories {
+  border: 1px solid red;
+  text-align: center;
+  font-size: 40px;
+  margin: 10px auto;
+  font-family: 'Cardinal', Arial, sans-serif;
+}
+
+.back-link {
+  border: 1px solid red;
+  text-align: center;
+  font-size: 40px;
+  margin: 10px auto;
+  font-family: 'Cardinal', Arial, sans-serif;
+}
+
+.add-to-cart {
+  border: 1px solid red;
+  text-align: center;
+  padding-top: 20px;
+  background: black;
+  opacity: 0.4;
+}
+
+.large-3.columns {
+  padding-top: 10px;
+  background: #5c5c5c;
+  border: 1px solid red;
+}
+
+.login-title {
+  border: 1px solid red;
+  text-align: center;
+  margin-top: 50px;
+  font-family: 'Cardinal', Arial, sans-serif;
+  font-size: 58px;
+  color: white;
+}
+
+.form {
+  border: 1px solid red;
+  margin-top: 30px;
+  color: white;
+}
+
+.cart h1 {
+  border: 1px solid red;
+  text-align: center;
+  margin-top: 50px;
+  font-family: 'SketchGothic', Arial, sans-serif;
+  font-size: 58px;
+  color: white;
+}
+
+.cart-item-total {
+  border: 1px solid red;
+  text-align: center;
+  font-size: 24px;
+  margin: 20px 0 10px 0;
+  font-family: 'Cardinal', Arial, sans-serif;
+}
+
+.cart-buttons {
+  border: 1px solid red;
+}
+
+h1.orders {
+  border: 1px solid red;
+  text-align: center;
+  margin-top: 50px;
+  font-family: 'SketchGothic', Arial, sans-serif;
+  font-size: 58px;
+  color: white;
+}
+
+table.orders{
+  border: 1px solid red;
+  margin: auto;
+}
+
+.order-title h1 {
+  border: 1px solid red;
+  text-align: center;
+  margin-top: 50px;
+  font-family: 'SketchGothic', Arial, sans-serif;
+  font-size: 58px;
+  color: white; 
+}
+
+.order-title p {
+  border: 1px solid red;
+  text-align: center;
+  margin-top: 50px;
+  font-family: 'Cardinal', Arial, sans-serif;
+  font-size: 36px;
+  color: white;
+}
+
+.order-total-price {
+  border: 1px solid red;
+  text-align: center;
+  margin: 10px auto;
+  font-family: 'Cardinal', Arial, sans-serif;
+  font-size: 24px;
+  color: white;
+}
+
+.order-status {
+  border: 1px solid red;
+  text-align: center;
+  margin: 10px auto;
+  font-family: 'Cardinal', Arial, sans-serif;
+  font-size: 24px;
+  color: white;
+}
+
+.order-submitted {
+  border: 1px solid red;
+  text-align: center;
+  margin: 10px auto;
+  font-family: 'Cardinal', Arial, sans-serif;
+  font-size: 24px;
+  color: white;
+}
+
+.order-updated {
+  border: 1px solid red;
+  text-align: center;
+  margin: 10px auto;
+  font-family: 'Cardinal', Arial, sans-serif;
+  font-size: 24px;
+  color: white;
+}
+
+.order-details {
+  border: 1px solid red;
+  background-color: #6E2602; 
+}
+
+.items-from-order {
+  margin-top: 20px;
+  border: 1px solid red;
+}
+
+.order-item {
+  border: 1px solid red;
+  background: black;
+  margin: 20px auto;
+}
+
+.order-item-title {
+  border: 1px solid red;
+  text-align: center;
```

Comment:
Properties should be ordered border, font-family, font-size, margin-top, text-align